docs/configuration/configuration.md

```diff
@@ -1467,6 +1467,7 @@ For OVHcloud's [public cloud instances](https://www.ovhcloud.com/en/public-cloud
 * `__meta_ovhcloud_dedicated_server_ipv6`: the IPv6 of the server
 * `__meta_ovhcloud_dedicated_server_link_speed`: the link speed of the server
 * `__meta_ovhcloud_dedicated_server_name`: the name of the server
+* `__meta_ovhcloud_dedicated_server_no_intervention`: is this server immune to datacenter intervention. Also see: [noIntervention](https://eu.api.ovh.com/console-preview/?section=%2Fdedicated%2Fserver&branch=v1#get-/dedicated/server/-serviceName-)
```

Comment:
```suggestion
* `__meta_ovhcloud_dedicated_server_no_intervention`: whether datacenter intervention is disabled.
```
WDYT?
I see, we already link the API above (users can look for details there.)

Reply:
This one looks better. Thank you.